requirements.adoc

```diff
@@ -272,20 +357,20 @@ ACCEPTANCE CRITERIA::
 
 * Verify that a referring physician can enter a new radiology order by entering
 ** patient
-** an orderable from concepts (i.e. "R/O pulmonary embolism")
+** an orderable from concepts eg CT head with contrast ~~(i.e. "R/O pulmonary embolism")~~
 ** urgency [choose from EMERGENCY, ROUTINE, ON SCHEDULED DATE;
 defaults to ROUTINE]
 ** date and time
 ** orderer from providers (in case I am not an RP, but a data clerk with
 RP permissions entering the order for an RP)
 ** order reason as free text
-** order reason from concept
+** order reason from concept eg ICD 9 diagnosis code
 ** a comment for the fulfiller of the order
 * Verify that a referring physician needs to enter a patient, orderable,
 urgency and orderer
 * Verify that a referring physician only needs to enter an order date and time
 if urgency is set to ON SCHEDULED DATE
-* Verify that a non-referring physician cannot create a radiology order
+* ~~Verify that a non-referring physician cannot create a radiology order~~  ### Judy : Not sure about this .. do you have a specific example?
```

Comment:
well maybe its not well put, I just mean a user who does not have permission to enter an order :)